src/components/common/UserDetails.tsx

```
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Box, Text, HStack, VStack } from '@chakra-ui/react';
-import { formatDate } from '../../utils/jsHelper/helper';
+import { formatDate, calculateAge, formatText } from '../../utils/jsHelper/helper';
```

ESLint

[error] Replace `·formatDate,·calculateAge,·formatText·` with `⏎↹formatDate,⏎↹calculateAge,⏎↹formatText,⏎`

(prettier/prettier)